app/hooks/useWalletTokens.ts

```diff
@@ -161,18 +176,18 @@ export const useWalletTokens = (walletsState: ConnectedWalletsState) => {
         };
       };
 
-      const fetchNativeTokens = async (poolAsset: string) => {
-        const chainKey = getChainKeyFromChain(poolAsset.split(".")[0]);
+      const fetchNativeTokens = async (pool: PoolDetail) => {
+        const chainKey = getChainKeyFromChain(pool.asset.split(".")[0]);
```

Comment:
We should use util fn `parseAssetString` to get the chain string.

Ideally parseAssetString should be something like this, but we don't have to make this change now --

```
/**
 * Parse asset details from asset string
 */
export const parseAssetString = ({ asset }: PoolDetail): [ChainType, string] => {
  invariant(asset, "asset must be non-null");
  const [chain = "", identifier = ""] = asset.split(".");
  const chainConfig = CHAINS.find(c => c.thorchainIdentifier === chain.toLowerCase())
  invariant(chainConfig, "pool is not supported by the application");
  return [chainConfig, identifier];
};
```